src/unit.rs

```diff
@@ -3808,6 +3812,7 @@ mod tests {
         where Endian: Endianity
     {
         let value = entry.attr_value(constants::DW_AT_name)
+            .expect("Should have parsed the name attribute")
```

Comment:
This is in a unit test. We use `expect` there instead of returning `Result`, and having separate expects lets us know exactly which one failed.